src/Schema/Events/Click.ts

```diff
@@ -522,6 +521,28 @@ export interface ClickedNavigationTab {
   subject: string
 }
 
+/**
+ *  User clicks in one of the price options on the offer page
+ *
+ *  This schema describes events sent to Segment from [[clickedOfferOption]]
+ *
+ *  @example
+ *  ```
+ *  {
+ *    action: "clickedOfferOption",
+ *    context_module: "OrdersOffer",
+ *    context_page_owner_type: "orders-offer",
+ *    context_page_owner_id: "dd0cbbb5-300b-4c49-92a1-fed55b077fa9"
+ *  }
+ * ```
+ */
+ export interface ClickedOfferOption {
+  action: ActionType.clickedOfferOption
+  context_module: ContextModule
```

Comment:
I actually don't think we need this. Context module is useful to locate a component on a page - here there is not much ambiguity.

Reply:
However, don't we need other properties? I think we talked about a property for which option the user select, and properties for amount/currency.

Reply:
I couldn't find anything similar, so I thought it would be only in the engineer side. I included, but I am not sure!